src/drawers/symbolDrawer.ts

```diff
@@ -23,13 +23,13 @@ export module _Drawer {
       delete attrToProjector["r"];
 
       attrToProjector["transform"] = (datum: any, index: number) =>
-        "translate(" + xProjector(datum, index) + "," + yProjector(datum, index) + ") " +
-        "scale(" + rProjector(datum, index) / 50 + ")";
+        "translate(" + xProjector(datum, index) + "," + yProjector(datum, index) + ")";
 
       var symbolProjector = attrToProjector["symbol"];
       delete attrToProjector["symbol"];
 
-      attrToProjector["d"] = symbolProjector;
+      attrToProjector["d"] = (datum: any, index: number) =>
```

Comment:
Shouldn't this occur in `generateAttrToProjector()` on `Plot.Scatter`?

For that matter, should we add ` = attrToProjector["d"] || ..." on the off-chance that some expert has decided to write a projector that directly returns path-strings?